src/test/regress/sql/multi_partitioning.sql

```diff
@@ -1127,6 +1127,23 @@ GROUP BY
 ORDER BY
 	1,2;
 
+
+-- test attaching and detaching partitions from partitioned tables with foreign keys
```

Comment:
These are to check we don't have a distributed deadlock in these cases. I should add an explicit comment for that.